### DISCUSSION TITLE
Has anyone got the SMTP settings to work with Exchange Online (Office365)

### DISCUSSION THREAD

**Discussion by vnetonline** [oqtane/oqtane.framework#3064] [Q&A]
It seems that TLS setting is missing because Exchange Online needs TLS 

`Error in processing. The server response was: 5.7.3 STARTTLS is required to send mail [SY5P282CA0024.AUSP282.PROD.OUTLOOK.COM 2023-07-26T05:24:50.176Z 08DB89382191A8E4]`

has anyone for Oqtane to work with Exchange Online (Office365)

**Comment by leigh-pointer** [comment]
@vnetonline 
Found this on the net
Microsoft 365 / Office 365 SMTP Settings?

SMTP Server: smtp.office365.com
Port: 587 (learn [more about SMTP ports](https://kinsta.com/blog/smtp-port/))
Requires SSL: Yes
Requires TLS: Yes (if available)
Authentication: Yes (choose Login if multiple options available)
Username: Your full Microsoft 365 email address (e.g. name@yourdomain.com)
Password: The password associated with your Microsoft 365 email address. You or your Microsoft 365 admin entered this password when creating the email account.

**Reply by vnetonline** [reply]
tried this not working 

`The SMTP server requires a secure connection or the client was not authenticated. The server response was: 5.7.57 Client not authenticated to send mail. Error: 535 5.7.3 Authentication unsuccessful [SY5P282CA0071.AUSP282.PROD.OUTLOOK.COM 2023-07-26T05:35:34.250Z 08DB8B4923C7A5F4]`

The username and password are right I checked and double checked then triple checked

**Comment by vnetonline** [comment]
got it to work ... the User Name and Email Sender has to be the same else it will not work
